package.json

```diff
@@ -77,7 +76,9 @@
     ],
     "moduleFileExtensions": [
       "ts",
-      "tsx"
+      "tsx",
+      "js",
```

Comment:
according to facebook/jest#6514, it also works for me, but i don't know why.

nowhere i depend source-map.js
which package depend source-map.js
only ts, why use modleFileExtensions ["js", "jsx"]